kolibri/plugins/learn/assets/src/views/SearchBox.vue

```diff
@@ -180,7 +180,6 @@
     display: table;
     width: 100%;
```

Comment:
One thing I see is that the outline overlays the clear (X) icon button. Not a blocking issue - but I think that setting the width like `width: calc(100% - 8px)` should put enough space there to avoid the overlap.